assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -77,6 +77,7 @@ function TextPane(props) {
         value={''}
         placeholder={__('Search', 'web-stories')}
         onChange={() => {}}
+        disabled={true}
```

Comment:
```suggestion
        disabled
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/975/commits/13691992cf61a8383f29d0ed470b0644954ee90b